src/camera.h

```diff
@@ -71,20 +73,25 @@ class camera
             if ((*world)->hit(cur_ray, hit_interval, rec)) {
                 ray scattered;
                 float3 attenuation; //means color
+                float3 color_from_emission = rec.mat_ptr->emitted();
                 if (rec.mat_ptr->scatter(cur_ray, rec, attenuation, scattered, local_rand_state)) {
-                    cur_attenuation *= attenuation;
+                    cur_attenuation *= attenuation + color_from_emission;
                     cur_ray = scattered;
                 }
                 else {
-                    return make_float3(0.0,0.0,0.0);
+                    cur_attenuation *= color_from_emission;
+                    return cur_attenuation;
+                    
+                    // return make_float3(0.0,0.0,0.0);

```

Comment:
Hmm I guess this can be removed, but let me know if I'm wrong :)

Reply:
no it cannot

Reply:
I'm talking about this line:
```
 // return make_float3(0.0,0.0,0.0);
```
If there is no information like "Comment on lines +91 to +94" this means that this comment refers to the first bottom line which is visible, I guess you could've miss that. Anyway, we don't need to remove that, we can get rid of it at a later point.